packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -18,6 +18,10 @@ import TimeSpacer from './_TimeSpacer';
 const cx = classNamesBind.bind(styles);
 
 const propTypes = {
+  /**
+   * String that labels the current element. 'aria-label' must be present for accessibility.
+   */
+  ariaLabel: PropTypes.string,
```

Comment:
This prop is not required you can pass value for exisiting `a11yLabel` from dateTimepicker.

Reply:
done https://github.com/cerner/terra-framework/pull/2031/commits/2d63b33d6e0de4b213da02c9b075de1cf5a96ae9